sql1.txt

```diff
@@ -5,25 +5,37 @@ SELECT * FROM Products LIMIT 5;
 
 1. Use an SQL SELECT statement to retrieve the first 10 rows of the Customer table, ordered by CustomerName.
 Paste your SQL statement below:
-
+SELECT * FROM customers
+ORDER by CustomerName ASC
+LIMIT 10; 
 
 2. Use an SQL Select statement to retrieve the names and prices for all products that cost less than $20.
 Paste your SQL statement below:
-
+SELECT ProductName, Price 
+FROM Products
+WHERE Price < 10;
```

Comment:
The structure of the query looks good, so I'm assuming that this was simply a typo:

```suggestion
WHERE Price < 20;
```